### PR TITLE
Remove projects references from sidebar and SNV page

### PR DESCRIPTION
**Pull request by inglor**
Signed-off-by: Leonidas Spyropoulos <artafinde@gmail.com>

### CODE REVIEW

File: templates/public/index.html [hunk 1]
```diff
@@ -157,8 +157,6 @@ <h4>Development</h4>
     <ul>
         <li><a href="https://wiki.archlinux.org/title/Getting_involved"
             title="Getting involved">Getting involved</a></li>
-        <li><a href="https://projects.archlinux.org/"
-            title="Official Arch projects (git)">Projects in Git</a></li>
```
Comment (jelly) [inline templates/public/index.html:161]: This should link to Gitlab

File: templates/public/svn.html [hunk 2]
```diff
@@ -7,13 +7,6 @@ <h2 class="title">SVN Repositories</h2>
     The PKGBUILD files can be fetched via the ABS utility. To learn more
     about ABS, see <a href="https://wiki.archlinux.org/title/ABS">the ABS wiki page</a>.
     </p>
-    <p>The SVN repositories have been cloned into git repositories and can be
-    viewed via the cgit interface.
-    <a href="https://projects.archlinux.org/svntogit/packages.git/">All
-        packages</a> are available here except for
-    <a href="https://projects.archlinux.org/svntogit/community.git/">community
-        and multilib</a> which are available in a different repository.</p>
-    <p>
```
Comment (jelly) [inline templates/public/svn.html:16]: Ideally this would link to our Github repository.

**Review by jelly — CHANGES_REQUESTED** [review]
Please resolve the typo in the commit message SNV => SVN and the comments.

**Comment by inglor** [comment]
See https://github.com/archlinux/archweb/pull/359

**Pull request closed without merging**